modules/byoh-configuring.adoc

```diff
@@ -16,6 +16,7 @@ Any Windows instances that are to be attached to the cluster as a node must fulf
 * The default shell for the SSH server must be the link:https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_server_configuration#configuring-the-default-shell-for-openssh-in-windows[Windows Command shell], or `cmd.exe`.
 * Port 10250 must be open for log collection.
 * An administrator user is present with the private key used in the secret set as an authorized SSH key.
+* If you are creating a BYOH Windows instance for an installer-provisioned infrastructure (IPI) AWS cluster, you must add a tag to the AWS instance that matches the `spec.template.spec.value.tag` value in the machine set for your worker nodes. For example, `kubernetes.io/cluster/<cluster_id>: owned` or `kubernetes.io/cluster/<cluster_id>: shared`.
```

Comment:
LGTM! This is a minor suggestion and you do not have to take it below. I wasn't sure if it would be better in this format. Either way it looks good! :) 
```suggestion
* If you are creating a BYOH Windows instance for an installer-provisioned infrastructure (IPI) AWS cluster, you must add a tag to the AWS instance that matches the `spec.template.spec.value.tag` value in the machine set for your worker nodes. 
** For example, `kubernetes.io/cluster/<cluster_id>: owned` or `kubernetes.io/cluster/<cluster_id>: shared`.
```